README.md

```diff
@@ -21,7 +21,12 @@ Currently has support for:
 
 ## Install ##
 - `pip install mppsolar` (minimal install), or
-- `pip install mppsolar[ble]` (install including BLE support aka jkbms), or
+- `pip install mppsolar[api]` (install server stuff), or
```

Comment:
I am not proud about "server stuff", so if you come up with something better, that would be appreciated :)